src/pages/index.js

```diff
@@ -262,14 +265,23 @@ const Index = ({ data: initialData }) => {
   useEffect(() => {
     setAdUnits(getAdUnits())
   }, [])
-
   // Only render ads if we are on the client side.
   const [shouldRenderAds, setShouldRenderAds] = useState(false)
   useEffect(() => {
     if (isClientSide()) {
       setShouldRenderAds(true)
     }
   }, [])
+  const { app, user } = data || {}
+  const [userGlobalId] = useState(get(user, 'id'))
```

Comment:
Sorry for not catching this in the first review—but I don't see any reason for this to live in state. Instead, just assign it to a `userGlobalId` variable.

Reply:
@kmjennison  will test again, but I thought I noticed a thing where even tho the value was staying the same, everytime we refetched data it would reassign that value and then recall the mutation

Reply:
Hmm, that would be unexpected. LMK if you can repro that, and I'd be interested in taking a look.

Reply:
wasn't able to reproduce, must have just been with the UUID which makes sense